packages/main/src/ListItemBase.js

```diff
@@ -68,15 +67,15 @@ class ListItemBase extends UI5Element {
 	}
 
 	_handleTabNext(event) {
-		const target = event.target.shadowRoot.activeElement;
```

Comment:
I think this should remain as is, maybe that's why the test is failing?

Reply:
Hi @vladitasev 
I had also a bad feeling when changing this, however the shadowRoot is not filled anymore when removing "_eventHandlersByConvention: true".

I will doublecheck that, thanks!